src/uct/api/uct.h

```diff
@@ -797,15 +797,15 @@ void uct_worker_progress_register_safe(uct_worker_h worker, ucs_callback_t func,
  * @ingroup UCT_CONTEXT
  * @brief Remove a slow path callback function from worker's progress.
  *
- * If 'id_p' points to a non-NULL handle, remove a callback which was previously
- * added by @ref uct_worker_progress_register_safe.
+ * If 'id_p' does not point to UCS_CALLBACKQ_ID_NULL, remove a callback which
+ * was previously added by @ref uct_worker_progress_register_safe.
  *
  * @param [in]    worker        Handle to the worker whose progress should invoke
  *                              the callback.
  * @param [inout] id_p          Points to callback identifier to remove, and filled
- *                              with NULL by this function. If *id_p already points
- *                              to NULL, no operation will be performed and *id_p
- *                              will be left unchanged.
+ *                              with UCS_CALLBACKQ_ID_NULL by this function. If *id_p
+ *                              already points to UCS_CALLBACKQ_ID_NULL, no operation
+ *                              will be performed and *id_p will be left unchanged.
```

Comment:
re-wording the id_p description for clarity:

Points to a callback identifier which indicates the callback to remove. If *id_p is not equal
to UCS_CALLBACKQ_ID_NULL, the callback will be removed and *id_p will be reset to 
UCS_CALLBACKQ_ID_NULL. If *id_p is equal to UCS_CALLBACKQ_ID_NULL, no operation
will be performed and *id_p will be left unchanged.